infrastructure/server-setup/tasks/docker.yml

```diff
@@ -41,14 +41,6 @@
     username: '{{dockerhub_username}}'
     password: '{{dockerhub_password}}'
 
-- name: 'Setup crontab to clean up docker images'
-  cron:
-    user: '{{ crontab_user }}'
-    name: 'cleanup docker images'
-    minute: '0'
-    hour: '0'
-    job: '/usr/bin/docker system prune -af >> /var/log/docker-prune.log'
-
```

Comment:
Think we need to keep this for a while and set it to `state: absent` instead - otherwise it only doesn't get installed to new environments but never gets removed from the existing environments

Reply:
ok i am changing it right now